docs/gettsim_objects/input_variables.md

```diff
@@ -496,3 +496,21 @@ Months of subjection to compulsory insurance in the 5 years before claiming unem
 insurance
 
 Type: float
+
+## `kind_unterh_brutto_m`
+
+Monthly gross child alimony payments
+
+Type: float
+
+## `kind_unterh_netto_m`
+
+Monthly actual child alimony payments
+
+Type: float
+
+## `partner_unterhalt_m`
```

Comment:
Just occurred to me while updating the description: Do we let this become negative in case we'll look at both payments made and received? Or should we think about more fine-grained cases?

Probably the same with kids above.

Reply:
Concerning this discussion: I found that the alimony payments paid might affect `arbeitsl_geld_2` as well as `wohngeld_m_hh` (potentially also `grunds_im_alter`) by reducing the considered income, see [SGBII](https://www.buzer.de/11b_SGB_II.htm) and [WoGG](https://www.wohngeld.org/wohngeldgesetz-wogg/paragraph18/). Thus, it might actually be necessary to have differing input variables depending on whether payments are made or received.

Reply:
For this discussion on alimony payments paid and not received, I just opened a new issue (#550) as I think this should be handled in a separate PR